### PR TITLE
Grafana and Prometheus

### PR DESCRIPTION
**Pull request by tbekas**

### CODE REVIEW

File: kubernetes/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    scrape_timeout: 10s
+    metrics_path: /metrics
+    scheme: http
+    static_configs:
+      - targets:
+          - localhost:9090
+  - job_name: dynamic-targets
+    http_sd_configs:
+      - url: http://l0-initial-validator:9000/targets
+      - url: http://l1-initial-validator:9000/target
```
Comment (marcinwadon) [inline kubernetes/prometheus/prometheus.yaml:23]: what's the difference between `targets` and `target`?

File: modules/core/src/main/scala/org/tessellation/modules/HttpApi.scala [hunk 2]
```diff
@@ -70,14 +72,16 @@ sealed abstract class HttpApi[F[_]: Async: SecurityProvider: KryoSerializer] pri
 
   private val debugRoutes = DebugRoutes[F](storages, services).routes
 
-  private val metricRoutes = MetricRoutes[F](services).routes
+  private val metricRoutes = routes.MetricRoutes[F]().routes
```
Comment (marcinwadon) [inline modules/core/src/main/scala/org/tessellation/modules/HttpApi.scala:75]: why `routes.` prefix is needed?

**Review by marcinwadon — DISMISSED** [review]
Looks fine, however `target` endpoint looks like typo

File: kubernetes/prometheus/prometheus.yaml (hunk 1 above)
Comment (tbekas) [reply kubernetes/prometheus/prometheus.yaml:23]: It should be `targets` on both, thanks for catching it, I changed it without checking if it's working.

**Review by kpudlik — DISMISSED** [review]
I like the fact that we got rid of ugly CRON ;d

File: kubernetes/prometheus/prometheus.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+scrape_configs:
+  - job_name: prometheus
+    scrape_interval: 15s
+    scrape_timeout: 10s
+    metrics_path: /metrics
+    scheme: http
+    static_configs:
+      - targets:
+          - localhost:9090
+  - job_name: dynamic-targets
```
[the remark this answers is not in the document]
Comment (tbekas) [reply kubernetes/prometheus/prometheus.yaml:20]: Thanks :)

**Review by marcinwadon — APPROVED** [review]

**Pull request merged**